assets/src/edit-story/elements/media/index.js

```diff
@@ -81,4 +81,5 @@ export const MEDIA_PANELS = [
   PanelTypes.LAYER_STYLE,
   PanelTypes.SIZE_POSITION,
   PanelTypes.ANIMATION,
+  PanelTypes.LINK,
```

Comment:
This is an unrelated change but just noticed while debugging to find the issue that the link panel is added to both elements separately as opposed to media generally.